apps/chat/src/components/Chat/TalkTo/TalkToSlider.tsx

```diff
@@ -304,16 +320,23 @@ export const TalkToSlider = ({
           }}
         >
           {sliderGroups.length ? (
-            sliderGroups.map((modelsGroup) => (
-              <SliderModelsGroup
+            sliderGroups.map((modelsGroup, idx) => (
```

Comment:
use `index` word instead
```suggestion
            sliderGroups.map((modelsGroup, index) => (
```